app/helpers/application_helper.rb

```diff
@@ -57,9 +57,9 @@ def custom_render_oembed_tag_async(document, canvas_index)
   # `iiif_canvas_id` and that `@id` conforms to the common DLSS `@id` format
   # e.g. https://purl.stanford.edu/ab123cd4567/iiif/canvas/ab123cd4567_1
   # @param [SirTrevorRails::Blocks::SolrDocumentsEmbedBlock] block
```

Comment:
I think the comment above is now invalid.

Reply:
:+1